MapofAccountNameAndId.cls

```diff
@@ -0,0 +1,38 @@
+public with sharing class MapofAccountNameAndId {
```

Comment:
Please check the Same Functionality in Changed Class name DisplayTheMapStructure